### ISSUE TITLE
License

### ISSUE THREAD

**Issue by jasonmhead** [bradfrost/frontend-guidelines-questionnaire#12]
Could this repo get a MIT or Apache2 license?
It could be useful for language model prompting.

**Comment by bradfrost** [comment]
@jasonmhead good idea! I don't have time to add an MIT license, but feel free to issue a PR.

**Comment by jasonmhead** [comment]
https://github.com/bradfrost/frontend-guidelines-questionnaire/pull/13
